Makefile

```diff
@@ -1,4 +1,10 @@
-.PHONY: all doc doc-man doc-latex docclean
+.PHONY: all allpf doc doc-man doc-latex docclean pfs
```

Comment:
allpf does not exist
target "doc-latex" is named "latex" below
pfs is a variable, i think there is no need to declare it as a phony target. maybe you were looking to $(pfs) expression evaluation at this point?